Testing/SystemTests/lib/systemtests/stresstesting.py

```diff
@@ -708,42 +742,123 @@ class TestManager(object):
     '''
 
     def __init__(self, test_loc, runner, output = [TextResultReporter()],
-                 testsInclude=None, testsExclude=None, exclude_in_pr_builds=None):
+                 quiet=False, testsInclude=None, testsExclude=None,
+                 exclude_in_pr_builds=None, showSkipped=False,
+                 output_on_failure=False, clean=False,
+                 test_count=[0,0,0], process_number=0, ncores=1):
```

Comment:
Can we avoid lists as default arguments as it's generally considered bad practice: https://docs.python-guide.org/writing/gotchas/#what-you-wrote